returnn/torch/data/pipeline.py

```diff
@@ -26,6 +26,7 @@
 import numpy
 import torch
 import torch.utils.data
+from torch.utils.data.dataset import T_co
```

Comment:
You don't really need that. That also causes a warning. It is not supposed to be imported.

Reply:
Oh. I think this was automatically added by the PyCharm auto generation. But agreed, this can be dropped.